contracts/libraries/Volatility.sol

```diff
@@ -99,9 +99,9 @@ library Volatility {
      */
     function amount0ToAmount1(uint128 amount0, int24 tick) internal pure returns (uint256 amount1) {
         uint160 sqrtPriceX96 = TickMath.getSqrtRatioAtTick(tick);
-        uint224 geometricMeanPriceX96 = uint224(FullMath.mulDiv(sqrtPriceX96, sqrtPriceX96, FixedPoint96.Q96));
```

Comment:
Contextually this is indeed geometric mean price. But _in theory_ this function could be used elsewhere, in which case it may be some other price